MapboxNavigation/RouteTableViewController.swift

```diff
@@ -85,7 +85,7 @@ class RouteTableViewController: UIViewController {
         }
     }
         
-    func notifyAlertLevelDidChange() {
+    func shouldGiveVoiceInstruction() {
```

Comment:
A table view controller has no reason to worry about spoken instructions. Rename this method to reflect what it does: `reload()`.